tests/signTransactionPage.test.ts

```diff
@@ -0,0 +1,325 @@
+    const invalidSecretKeyErrorMsg = page.getByText(
+      "Invalid secret key. Please check your secret key and try again.",
+    );
+    const successSecretKeyMsg = secretKeysView.getByText(
+      "Successfully added 1 signature",
+    );
+
+    await expect(multiPickerInput).toHaveCount(1);
+    await addAddlSecretBtn.click();
+    await expect(multiPickerInput).toHaveCount(2);
+
+    // Type in a string in an invalid secret key format
+    await multiPickerInput.nth(0).fill("lkjlsdkjflksdjf");
+    await expect(invalidSecretKeyErrorMsg).toBeVisible();
+
+    // Type in a string in an valid secret key format
+    await multiPickerInput
+      .nth(0)
+      .fill("SCAM6CZNCLJFQOGSC7LLE2KMBYCBD7S5IYV447MZX5NHPGCHRHPYITCF");
+    await expect(invalidSecretKeyErrorMsg).toBeHidden();
+    await expect(secretKeysSignBtn).toBeEnabled();
+
+    await secretKeysSignBtn.click();
+
+    await expect(successSecretKeyMsg).toBeVisible();
+
+    const firstSignedResponse = await signedXdr.getByText(
+      "AAAAAgAAAADJrq4b4AopDZibkeBWpDxuWKUcY4FUUNQdIEF3Nm9dkQAAAGQAAAIiAAAAAQAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAQAAAACXlGN76T6NQcaUJxbEkH3mi1HHWsHnLqMDdlLl9NlJgQAAAAAAAAAABfXhAAAAAAAAAAABjc9OtQAAAEBrpI8Q90yqEqjcLSubVj5nqtyt53bpVzi8Bzikps4xuom0xHQgrM6MsQS503ortwLcYOw0gyLPyst7J88ZDoQJ",
+    );
+
+    await expect(firstSignedResponse).toBeVisible();
+
+    /*** Signatures: Hardware Wallet ***/
+    const bipPathInput = await hardwareView.getByPlaceholder(
+      "BIP path in format: 44'/148'/0'",
+    );
+
+    await expect(bipPathInput).toHaveValue("44'/148'/0'");
+
+    const hardwareSelect = hardwareView.locator("id=hardware-wallet-select");
+
+    await hardwareSelect.click();
+
+    const options = await hardwareSelect.locator("option").allTextContents();
+    expect(options).toEqual([
+      "Select operation type",
+      "Ledger",
+      "Hash with Ledger",
+      "Trezor",
+    ]);
+
+    const hardwareSignBtn = hardwareView.getByText("Sign transaction");
+
+    await hardwareSignBtn.isDisabled();
+
+    await hardwareSelect.selectOption("Trezor");
+
+    await expect(hardwareSelect).toHaveValue("trezor");
+
+    await hardwareSignBtn.isEnabled();
+
+    await hardwareSelect.selectOption("Ledger");
+
+    await expect(hardwareSelect).toHaveValue("ledger");
+
+    await hardwareSelect.selectOption("Hash with Ledger");
+
+    await expect(hardwareSelect).toHaveValue("ledger_hash");
+
+    /*** Signatures: Wallet Extension ***/
+    const walletExtSignBtn = walletExtView.getByRole("button", {
+      name: "Sign with wallet",
+    });
+    await walletExtSignBtn.click();
+
+    await expect(
+      page.getByRole("heading", { name: "Connect a Wallet" }),
+    ).toBeVisible();
+
+    // Wallet Extension to display 6 wallets
+    await expect(page.getByRole("listitem")).toHaveCount(6);
+
+    // Exit out of the wallet extension modal
+    await page.click("body", { position: { x: 10, y: 10 } });
```

Comment:
This test will break if the close button position changes, but style changes shouldn't affect the tests. We can't use a class selector? Something like `page.locator(".wallets-header__button")`.

Reply:
it's more of a clicking outside of the modal